frontend/docs/scripts/deploy-s3.js

```diff
@@ -0,0 +1,43 @@
+const gulp = require('gulp');
+const merge = require('merge-stream');
+const awspublish = require('gulp-awspublish');
+const AWS = require('aws-sdk');
+
+const deploy = publisher => {
+  const zippedFiles = [
+    './build/**/*.svg',
+    './build/**/*.html',
+    './build/**/*.css',
+    './build/**/*.js',
+    './build/**/*.map',
+    './build/**/*.jpg'
+  ];
+  const unzippedFiles = ['./build/**/*'].concat(zippedFiles.map(file => `!${file}`));
+  const headers = {
+    'Content-Encoding': 'gzip'
+  };
+
+  const zipped = gulp.src(zippedFiles).pipe(awspublish.gzip({ ext: '' })).pipe(publisher.publish(headers));
+  const unzipped = gulp.src(unzippedFiles).pipe(publisher.publish());
+  return merge(zipped, unzipped).pipe(publisher.sync()).pipe(awspublish.reporter());
+};
+
+gulp.task('publish-stg', () => {
```

Comment:
I'd extract this arrow function inside a parametrizable function, so we can reuse it in `publish-prod` task.